File: Harvard/CS50 Intro to Computer Science/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void) 
{
    int height;
    
    do 
    {
        height = get_int("Height: ");
    }
    while (height <= 0 ||  height > 8);
    // i accounts for each row
    for (int i = 1; i <= height; i++) 
    {
        // j accounts for the spaces before left side
        for (int j = height - i; j != 0; j--)
        {
            printf(" ");
        }
        // x accounts for the left hashes
        for (int x = i; x != 0; x--)
        {
            printf("#");
        }
        // accounts for the constant double space
        printf("  ");
        
        // account for the right hashes
        for (int y = i; y != 0; y--) 
        {
            printf("#");
        }
        // goes to the next row
        printf("\n");
    }
    
}

    

File: Harvard/CS50 Intro to Computer Science/pset6/mario/mario.py
import cs50

height = 0

while (height <= 0 or height >= 9):
    height = cs50.get_int('height: ')    

i = height

while(i > 0):
    
    j = height - i + 1
    x = i - 1
    
    while(x > 0):
        print(" ", end="")
        x -= 1
    
    while(j > 0):
        print("#", end="")
        j -= 1
        
    print("  ", end="")
    
    j = height - i + 1
    
    while (j > 0):
        print("#", end="")
        j -= 1
        
    i -= 1
    
    print()